cuda_core/pixi.toml

```diff
@@ -0,0 +1,129 @@
+# *after* the package is built, and the Cython tests depend on CUDA headers
+# transitively (through cuda-bindings)
+cuda-cudart-dev = "*"
+cuda-profiler-api = "*"
+cuda-nvrtc-dev = "*"
+
+[feature.cython-tests.target.linux-64.dependencies]
+cuda-crt-dev_linux-64 = "*"
+
+[feature.cython-tests.target.linux-aarch64.dependencies]
+cuda-crt-dev_linux-aarch64 = "*"
+
+[feature.cython-tests.target.win-64.dependencies]
+cuda-crt-dev_win-64 = "*"
+
+# For finding headers when building the Cython tests
+[feature.cython-tests.target.linux-64.activation.env]
+CUDA_HOME = "$CONDA_PREFIX/targets/x86_64-linux"
+
+[feature.cython-tests.target.linux-aarch64.activation.env]
+CUDA_HOME = "$CONDA_PREFIX/targets/sbsa-linux"
+
+[feature.cython-tests.target.win-64.activation.env]
+CUDA_HOME = '%CONDA_PREFIX%\Library'
+
+[feature.cu13.system-requirements]
+cuda = "13"
+
+[feature.cu13.dependencies]
+cuda-version = "13.*"
+
+[feature.cu12.system-requirements]
+cuda = "12"
+
+[feature.cu12.dependencies]
+cuda-version = "12.*"
+
+# We keep both cu12 and cu13 because cuda.core works with either major version
+[environments]
+cu13 = { features = ["cu13", "test", "cython-tests"], solve-group = "cu13" }
+cu12 = { features = ["cu12", "test", "cython-tests"], solve-group = "cu12" }
+
+# TODO: check if these can be extracted from pyproject.toml
+[package]
+name = "cuda-core"
+version = "0.4.1"
+
+[package.build]
+backend = { name = "pixi-build-python", version = "*" }
+
+[package.build.config]
+env = { CUDA_PYTHON_PARALLEL_LEVEL = "$(nproc)" }
+noarch = false
+compilers = ["c", "cxx"]
+
+[package.build.target.linux-64.config.env]
+CUDA_HOME = "$PREFIX/targets/x86_64-linux"
+
+[package.build.target.linux-aarch64.config.env]
+CUDA_HOME = "$PREFIX/targets/sbsa-linux"
+
+[package.build.target.win-64.config.env]
+CUDA_HOME = '%PREFIX%\Library'
+
+# TODO: revisit this
+# [package.build-dependencies]
+# ccache = "*"
+
+[package.host-dependencies]
+python = "*"
+pip = "*"
+setuptools = ">=77"
+cython = ">=3.1"
+cuda-cudart-dev = "*"
+cuda-profiler-api = "*"
+# TODO: figure out why we can't set editable=true here
+cuda-bindings = { path = "../cuda_bindings" }
+
+[package.target.linux-64.host-dependencies]
+cuda-crt-dev_linux-64 = "*"
+
+[package.target.linux-aarch64.host-dependencies]
+cuda-crt-dev_linux-aarch64 = "*"
+
+[package.target.win-64.host-dependencies]
+cuda-crt-dev_win-64 = "*"
+
+[package.run-dependencies]
+python = "*"
+numpy = "*"
+# TODO: figure out why we can't set editable=true here
+cuda-bindings = { path = "../cuda_bindings" }
+
+[target.linux.tasks.build-cython-tests]
+cmd = ["$PIXI_PROJECT_ROOT/tests/cython/build_tests.sh"]
+
+[target.linux.tasks.run-cython-tests]
+cmd = ["pytest", "$PIXI_PROJECT_ROOT/tests/cython/"]
+depends-on = [{ task = "build-cython-tests" }]
+
+[target.linux.tasks.test]
+cmd = ["pytest", "$PIXI_PROJECT_ROOT/tests"]
+#env = { PYTHONPATH = "$PIXI_PROJECT_ROOT/../cuda_python_test_helpers" }
```

Comment:
This does not seem necessary...? (I commented it out.)

Reply:
Running tests from `$PIXI_PROJECT_ROOT` doesn't work for me without this. That's where we should run the tests from.

This was an intentional choice that I made so that `pixi run -e cu13 pytest` "just works".